renderer/views/player.js

```diff
@@ -116,6 +117,7 @@ function renderPlayerControls (state, dispatch) {
 
   // Handles a click or drag to scrub (jump to another position in the video)
   function handleScrub (e) {
+    dispatch('videoMouseMoved')
```

Comment:
@dcposch That's what I had expected too, but the mousemove event is attached to .player and .letterbox. When you're scrubbing on .scrub-bar, it doesn't register the mousemove event. Try dragging the scrub bar for more than two seconds, it will hide.

Also, in my tests, adding another mousemove event to the div.scrub-bar itself does not execute when the user is dragging. I believe that might be because it's relative to the element itself, so if you're dragging, the cursor is actually going to stay in the same position relative to the scrub-bar.